source/common/network/listener_filter_buffer_impl.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <bits/stdint-uintn.h>
```

Comment:
Is this needed? I don't think this is portable, and I would be surprised if the definition of uint8_t isn't already included.

Reply:
Looks like everytime I use uint*_t, the vscode add it. But yes, the code works without this include.